### ISSUE TITLE
Assertion `0 && "const fp bug in interp one"' failed.

### ISSUE THREAD

**Issue by kandy** [facebook/hhvm#4981]
Host: hhvm
ProcessID: 23112
ThreadID: 7fadf463d500
ThreadPID: 23112
Name: /usr/bin/hhvm
Type: Aborted
Runtime: hhvm
Version: heads/master-0-g90f4b2dba8ad7b0ffca9dd92cd9d38b3da8b55dd
DebuggerCount: 0

Arguments: setup/index.php install --db_host=localhost --db_name=magento2sample --db_user=root --backend_frontname=backend --admin_username=admin --admin_password=123123q --admin_email=admin@email.local --admin_firstname=First --admin_lastname=Last --base_url=http://magento2.tk/ --base_url_secure=http://magento2.tk/ --language=en_US --currency=USD --use_rewrites=true --use_secure=false --use_secure_admin=false
ThreadType: CLI

```
# 0  ?? at hhvm:0
# 1  __GI_raise at /build/buildd/eglibc-2.19/signal/../nptl/sysdeps/unix/sysv/linux/raise.c:56
# 2  __GI_abort at /build/buildd/eglibc-2.19/stdlib/abort.c:91
# 3  __assert_fail_base at /build/buildd/eglibc-2.19/assert/assert.c:92
# 4  __assert_fail at /lib/x86_64-linux-gnu/libc.so.6:0
# 5  HPHP::jit::x64::CodeGenerator::cgInterpOneCommon(HPHP::jit::IRInstruction*) at hhvm:0
# 6  HPHP::jit::x64::CodeGenerator::cgInterpOneCF(HPHP::jit::IRInstruction*) at hhvm:0
# 7  HPHP::jit::x64::CodeGenerator::cgInst(HPHP::jit::IRInstruction*) at hhvm:0
# 8  ?? at hhvm:0
# 9  HPHP::jit::x64::BackEnd::genCodeImpl(HPHP::jit::IRUnit&, HPHP::jit::AsmInfo*) at hhvm:0
# 10 HPHP::jit::genCode(HPHP::jit::IRUnit&) at hhvm:0
# 11 HPHP::jit::MCGenerator::traceCodeGen(HPHP::jit::HTS&) at hhvm:0
# 12 ?? at hhvm:0
# 13 HPHP::jit::translateRegion(HPHP::jit::HTS&, HPHP::jit::RegionDesc const&, HPHP::hphp_hash_set<HPHP::jit::ProfSrcKey, HPHP::jit::ProfSrcKey::Hasher, std::equal_to<HPHP::jit::ProfSrcKey> >&, HPHP::jit::TransFlags, std::vector<HPHP::jit::RegionDesc::TypePred, std::allocator<HPHP::jit::RegionDesc::TypePred> >&) at hhvm:0
# 14 HPHP::jit::MCGenerator::translateWork(HPHP::jit::TranslArgs const&) at hhvm:0
# 15 HPHP::jit::MCGenerator::translate(HPHP::jit::TranslArgs const&) at hhvm:0
# 16 HPHP::jit::MCGenerator::retranslate(HPHP::jit::TranslArgs const&) at hhvm:0
# 17 HPHP::jit::MCGenerator::createTranslation(HPHP::jit::TranslArgs const&) at hhvm:0
# 18 HPHP::jit::MCGenerator::getTranslation(HPHP::jit::TranslArgs const&) at hhvm:0
# 19 HPHP::jit::MCGenerator::getCallArrayPrologue(HPHP::Func*) at hhvm:0
# 20 HPHP::jit::funcBodyHelper(HPHP::ActRec*, void*) at hhvm:0
# 21 HPHP::jit::x64::BackEnd::enterTCHelper(unsigned char*, HPHP::ActRec*) at hhvm:0
# 22 HPHP::jit::MCGenerator::enterTC(unsigned char*, HPHP::ActRec*) at hhvm:0
# 23 HPHP::jit::MCGenerator::enterTCAfterPrologue(unsigned char*) at hhvm:0
# 24 ?? at hhvm:0
# 25 ?? at hhvm:0
# 26 HPHP::ExecutionContext::invokeFunc(HPHP::TypedValue*, HPHP::Func const*, HPHP::Variant const&, HPHP::ObjectData*, HPHP::Class*, HPHP::VarEnv*, HPHP::StringData*, HPHP::ExecutionContext::InvokeFlags) at hhvm:0
# 27 HPHP::ExecutionContext::invokeUnit(HPHP::TypedValue*, HPHP::Unit const*) at hhvm:0
# 28 ?? at hhvm:0
# 29 ?? at hhvm:0
# 30 HPHP::include_impl_invoke(HPHP::String const&, bool, char const*) at hhvm:0
# 31 HPHP::hphp_invoke(HPHP::ExecutionContext*, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool, HPHP::Array const&, HPHP::VRefParamValue const&, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool&, std::basic_string<char, std::char_traits<char>, std::allocator<char> >&, bool, bool, bool) at hhvm:0
# 32 HPHP::hphp_invoke_simple(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool) at hhvm:0
# 33 ?? at hhvm:0
# 34 HPHP::execute_program(int, char**) at hhvm:0
# 35 main at hhvm:0
# 36 __libc_start_main at /build/buildd/eglibc-2.19/csu/libc-start.c:321
# 37 ?? at hhvm:0
```

**Comment by fredemmott** [comment]
Possibly fixed by b360c847e54be913f004ee009e244164bc33ed0f

**Comment by fredemmott** [comment]
Can you retest with that fix in?

**Comment by kandy** [comment]
fixed

**Comment by rrh** [comment]
I concur that this appears to be fixed by  fixed by b360c84
